modular_skyrat/modules/loadouts/loadout_items/under/loadout_datum_under.dm

```diff
@@ -677,3 +440,281 @@ GLOBAL_LIST_INIT(loadout_miscunders, generate_loadout_items(/datum/loadout_item/
 	name = "Swept Skirt"
 	item_path = /obj/item/clothing/under/dress/skirt/swept
 
+
+/datum/loadout_item/under/miscellaneous/gear_harness
+	name = "gear harness"
+	item_path = /obj/item/clothing/under/misc/gear_harness
+
+/datum/loadout_item/under/miscellaneous/kimunder
+	name = "gear harness"
+	item_path = /obj/item/clothing/under/misc/kimunder
+
+/datum/loadout_item/under/miscellaneous/countess
+	name = "countess dress"
+	item_path = /obj/item/clothing/under/misc/countess
+
+/datum/loadout_item/under/miscellaneous/peakyblinder
+	name = "birmingham bling"
+	item_path = /obj/item/clothing/under/misc/peakyblinder
+
+/datum/loadout_item/under/miscellaneous/taccas
+	name = "tacticasual uniform"
+	item_path = /obj/item/clothing/under/misc/taccas
+
+/datum/loadout_item/under/miscellaneous/rancher
+	name = "rancher outfit"
+	item_path = /obj/item/clothing/under/rancher
+
+/datum/loadout_item/under/miscellaneous/rancher_pioneer
+	name = "pioneer outfit"
+	item_path = /obj/item/clothing/under/rancher/pioneer
+
+/datum/loadout_item/under/miscellaneous/rancher_worker
+	name = "western worker outfit"
+	item_path = /obj/item/clothing/under/rancher/worker
+
+/datum/loadout_item/under/miscellaneous/rancher_cowboy
+	name = "cowboy outfit"
+	item_path = /obj/item/clothing/under/rancher/cowboy
+
+/datum/loadout_item/under/miscellaneous/rancher_checkered
+	name = "western checkered outfit"
+	item_path = /obj/item/clothing/under/rancher/checkered
+
+/datum/loadout_item/under/miscellaneous/cargo_casual
+	name = "cargo tech casual wear"
+	item_path = /obj/item/clothing/under/rank/cargo/casualman
+	restricted_roles = list("Cargo Technician")
+
+/datum/loadout_item/under/miscellaneous/cargo_casual
```

Comment:
this and the leaves are meant to be contraband, as I don't think people should be running around with syndi looking gear...or naked, but you do you, your choice

Reply:
(that is, the cargo with evil in the path)

Reply:
Hmm, why are the leafs contraband t

Reply:
because hydro needed contraband- i mean

uhh...just weird to be wearing leafs, but again, you do you boss